adagucserverEC/CCairoPlotter.h

```diff
@@ -123,6 +125,8 @@ class CCairoPlotter {
   void writeToPng32Stream(FILE *fp, unsigned char alpha);
   void writeToWebP32Stream(FILE *fp, unsigned char alpha, int quality);
   void setToSurface(cairo_surface_t *png);
+  void drawStrokedText(int x, int y, double angle, const char *text, const char *fontFile, float fontSize, float strokeWidth, CColor bgcolor, CColor fgcolor);
+  void drawBarb(int x, int y, double direction, double strength, CColor color, CColor outlineColor, bool drawOutline, float lineWidth, bool toKnots, bool flip);
```

Comment:
Move these a couple of lines up to directly below drawText function?

Reply:
Done